apis/sgv2-graphqlapi/src/test/java/io/stargate/sgv2/graphql/integration/cqlfirst/ApolloIntegrationTest.java

```diff
@@ -24,50 +24,59 @@
 import com.example.graphql.client.betterbotz.type.ProductsInput;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.TestProfile;
+import io.stargate.sgv2.common.testprofiles.IntegrationTestProfile;
+import io.stargate.sgv2.graphql.integration.util.ApolloIntegrationTestBase;
+import io.stargate.sgv2.graphql.integration.util.CqlFirstClient;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
-import net.jcip.annotations.NotThreadSafe;
+import javax.enterprise.context.control.ActivateRequestContext;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
 
 /**
  * Covers the CQL-first API using the apollo-runtime client library.
  *
  * <p>Please do not add new test methods here. We are trying to limit usage of apollo-runtime to
  * only a few tests because it requires too much boilerplate. Instead, create new test classes and
- * use {@link CqlFirstClient} (see {@link SelectTest}, {@link InsertTest}, etc).
+ * use {@link CqlFirstClient} (see {@link SelectIntegrationTest}, {@link InsertIntegrationTest},
+ * etc).
  *
  * <p>To update the existing methods here:
  *
  * <ul>
- *   <li>If the schema has changed, update `src/main/graphql/betterbotz/schema.json`. You can use
- *       the query in `src/main/resources/introspection.graphql` (paste it into the graphql
+ *   <li>If the schema has changed, update `src/test/graphql/betterbotz/schema.json`. You can use
+ *       the query in `src/test/resources/introspection.graphql` (paste it into the graphql
  *       playground at ${STARGATE_HOST}:8080/playground).
  *   <li>If there are new operations, create corresponding descriptors in
- *       `src/main/graphql/betterbotz`.
+ *       `src/test/graphql/betterbotz`.
  *   <li>Run the apollo-client-maven-plugin, which reads the descriptors and generates the
- *       corresponding Java types: `mvn generate-sources` (an IDE rebuild should also work). You can
- *       see generated code in `target/generated-sources/graphql-client`.
+ *       corresponding Java types: `mvn generate-test-sources` (an IDE rebuild should also work).
+ *       You can see generated code in `target/generated-test-sources/graphql-client`.
  * </ul>
  */
-@NotThreadSafe
-public class ApolloTest extends ApolloTestBase {
+@QuarkusIntegrationTest
+@TestProfile(IntegrationTestProfile.class)
+@ActivateRequestContext
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ApolloIntegrationTest extends ApolloIntegrationTestBase {
 
   @Test
   public void getOrdersByValue() throws ExecutionException, InterruptedException {
-    ApolloClient client = getApolloClient("/graphql/" + keyspace);
+    ApolloClient client = getApolloClient("/graphql/" + keyspaceId.asInternal());
```

Comment:
maybe `keyspaceId.asInternal()` should be a method in the `CqlFirstIntegrationTest`

Reply:
I'm not sure how best to explain it, but I think exposing the `CqlIdentifier` is better. That's the thing we want to make available; whether subclasses use the internal form or the CQL form is up to their use case.

Reply:
Fine with me